e2e/validation.go

```diff
@@ -34,7 +34,7 @@ func ValidateWASM(ctx context.Context, s *Scenario, nodeName string) {
 }
 
 func ValidateCommonLinux(ctx context.Context, s *Scenario) {
-	execResult := execOnVMForScenarioValidateExitCode(ctx, s, "cat /etc/default/kubelet", 0, "could not read kubelet config")
+	execResult := execOnVMForScenarioValidateExitCode(ctx, s, "sudo cat /etc/default/kubelet", 0, "could not read kubelet config")
```

Comment:
sorry I'm probably missing something obvious, but why do we need the `sudo` everywhere now? I think it's fine to use in testing contexts, though I'm mainly just wondering why we need it now

Reply:
I think currently it implicitly happens everywhere. I assume windows doesn't like it.
So the change is to call it explicitly everywhere now to allow windows behave differently.

Reply:
Yeah, by default we were adding 'sudo" to the start of every command executed on a node as part of a test. Given windows uses powershell, adding "sudo" to the command just won't work. It should be the decision of the test that creates the command anyway.